src/include/miopen/conv/problem_description.hpp

```diff
@@ -338,8 +338,10 @@ struct ProblemDescription
     }
     bool IsInt8() const
     {
-        return GetInDataType() == miopenInt8 && GetWeightsDataType() == miopenInt8 &&
-               (GetOutDataType() == miopenInt32 || GetOutDataType() == miopenFloat);
+        return (GetInDataType() == miopenInt8 && GetWeightsDataType() == miopenInt8 &&
+                (GetOutDataType() == miopenInt32 || GetOutDataType() == miopenFloat)) ||
+               (GetOutDataType() == miopenInt8 && GetWeightsDataType() == miopenInt8 &&
+                (GetInDataType() == miopenInt32 || GetInDataType() == miopenFloat));
```

Comment:
This is misleading. Basically, IsIntTYPE should return true only when all data types are TYPE. This was acceptable as long as we only supported FWD INT8. Now it's time to put things in order.

Reply:
Hi Artem. We support FWD & BWD INT8 conv, but inference kernel is not ready, which means we have no way to make a INT8 output ( I mean output tensor type in FWD, input tensor type in BWD).
Shall we change the function name to `IsInt8_tmp()` or something else?